README.md

```diff
@@ -127,11 +127,14 @@ For Ubuntu or Debian Linux, the `master` and `release` branches are packaged and
 To install release builds via Apt:
 
 ```bash
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "8756 C4F7 65C9 AC3C B6B8  5D62 379C E192 D401 AB61"
```

Comment:
This is Bintray's key.  If we do start using another, we should tell people to install it, instead.